README.md

```diff
@@ -16,12 +25,13 @@ Debian: `sudo apt-get install python3`
 
 ## Usage
 
-You can double click the ease_generator.pyw file
+Double click the ease_generator.pyw file to run the program.
 
+* Make the necessary settings for fps, duration (in seconds), start and end position and of course select the type of easing
 * Click on "Generate and copy".
 * Go in kdenlive
 * Add a transform effect to a clip.
 * In the transform effect, Click the hamburger menu button left to the timestamp.
 * Select "Import Keyframes from Clipboard..."
 * Change the time offset (if necessary)
-* Disable "Limit keyframe number"
\ No newline at end of file
+* Disable "Limit keyframe number"
```

Comment:
what is changed here?

Reply:
I don't remember changing anything here. Maybe there was a the line break at the end of the file that was deleted?